package.json

```diff
@@ -31,7 +31,7 @@
     "prettier": "^2.5.1"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "react-scripts --openssl-legacy-provider start",
```

Comment:
Although this is a draft PR I'm curious. Is this required?

Reply:
The app won't work on my system without it.